src/pages/case/cleanio.js

```diff
@@ -14,7 +14,6 @@ const ProjectCover = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: center;
-  margin-top: 126px;
```

Comment:
good catch

Reply:
since we are repeating these cases components let's extract to a single component to be imported on every case page?

Reply:
can be!!! I used separated files thinking in different ways of expose what we do in the project, but, really, we aren't using this now